selectolax/lexbor/node.pxi

```diff
@@ -655,6 +655,57 @@ cdef class LexborNode:
         else:
             raise SelectolaxError("Expected a string or LexborNode instance, but %s found" % type(value).__name__)
 
+    def insert_child(self, str_or_LexborNode value):
```

Comment:
For simplicity I implemented `insert_child` the same way as surrounding code (e.g. `insert_after`), and just updated the docstring and function calls.